### ISSUE TITLE
Proposal: Move most of the barcode subclasses out in to a domain ontology

### ISSUE THREAD

**Issue by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#561]
These are too specific for mid-level. 

Keep only Barcode, One-Dimensional-Barcode and Two-Dimensional-Barcode

**Comment by CarterBeauBenson** [comment]
I agree, but much like the case with the familial relationships, where should they go? They should remain open sourced, but our domain extension opensource module selection is sparse.

**Comment by CarterBeauBenson** [comment]
If they are to move to their own module, do you see other classes that should move with them? Otherwise, it might be module with 20 IBA classes

**Comment by alanruttenberg** [comment]
I think it's fine to be a module with just 20 **ICE** classes. These become ICEs as part of #564
